lib/bibdata_rs/src/ephemera/ephemera_folder_builder.rs

```diff
@@ -134,12 +136,12 @@ impl EphemeraFolderBuilder {
             format: self.format,
             id,
             language: self.language,
-            origin: self.origin_place,
+            origin: self.origin_place.map(|v| VecSafe::from(VecSafe(v))),
```

Comment:
Clippy recommended for me:
```suggestion
            origin: self.origin_place.map(|v| VecSafe(v)),
```